src/style.scss

```diff
@@ -1,197 +1,203 @@
+    }
+  }
+
+  .title {
+    position: absolute;
+    z-index: 1;
+    color: #fff;
+    bottom: 20px;
+    left: 20px;
+    transition: opacity .8s cubic-bezier(0, 0.02, 0, 1);
+    font-size: clamp(20px, 3vw, 30px);
+    text-shadow: 0 4px 4px rgba(0, 0, 0, .1);
+  }
+
+  .num {
+    position: absolute;
+    z-index: 1;
+    color: #fff;
+    top: 10px;
+    left: 20px;
+    transition: opacity .8s cubic-bezier(0, 0.02, 0, 1);
+    font-size: clamp(20px, 10vw, 80px);
+  }
+
+  img {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+    pointer-events: none;
+  }
  }
-}
 
-.layout {
- position: absolute;
- z-index: 0;
- top: 0;
- left: 0;
- width: 100%;
- height: 100%;
- pointer-events: none;
-
- &:before {
-   content: '';
-   position: absolute;
-   z-index: 1;
-   top: 0;
-   left: 90px;
-   width: 10px;
-   height: 100%;
-   border: 1px solid #fff;
-   border-top: none;
-   border-bottom: none;
-   opacity: .15;
+ .layout {
+  position: absolute;
+  z-index: 0;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  pointer-events: none;
+
+  &:before {
+    content: '';
+    position: absolute;
+    z-index: 1;
+    top: 0;
+    left: 90px;
+    width: 10px;
+    height: 100%;
+    border: 1px solid #fff;
+    border-top: none;
+    border-bottom: none;
+    opacity: .15;
+  }
+
+  .box {
+    position: absolute;
+    bottom: 0;
+    left: 30px;
+    color: #fff;
+    transform-origin: 0% 10%;
+    transform: rotate(-90deg);
+    font-size: 9px;
+    line-height: 1.4;
+    text-transform: uppercase;
+    opacity: .4;
+  }
  }
 
- .box {
-   position: absolute;
-   bottom: 0;
-   left: 30px;
-   color: #fff;
-   transform-origin: 0% 10%;
-   transform: rotate(-90deg);
-   font-size: 9px;
-   line-height: 1.4;
-   text-transform: uppercase;
-   opacity: .4;
+ .logo {
+  position: absolute;
+  z-index: 2;
+  top: 28px;
+  right: 28px;
+  width: 30px;
+  height: 30px;
+  background: #fff;
+  border-radius: 50%;
+  opacity: .5;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-family: 'Orelo-sw-db', serif;
+  pointer-events: all;
+  color: black;
+  text-decoration: none;
+  font-size: 20px;
+  overflow: hidden;
+  padding-bottom: .1em;
  }
-}
-
-.logo {
- position: absolute;
- z-index: 2;
- top: 28px;
- right: 28px;
- width: 30px;
- height: 30px;
- background: #fff;
- border-radius: 50%;
- opacity: .5;
- display: flex;
- justify-content: center;
- align-items: center;
- font-family: 'Orelo-sw-db', serif;
- pointer-events: all;
- color: black;
- text-decoration: none;
- font-size: 20px;
- overflow: hidden;
- padding-bottom: .1em;
-}
 
-.social {
- position: absolute;
- z-index: 10;
- bottom: 20px;
- right: 25px;
- color: #fff;
- opacity: .4;
-
- a {
-   display: inline-block;
-   margin-left: 3px;
+ .social {
+  position: absolute;
+  z-index: 10;
+  bottom: 20px;
+  right: 25px;
+  color: #fff;
+  opacity: .4;
+
+  a {
+    display: inline-block;
+    margin-left: 3px;
+  }
+
+  svg {
```

Comment:
I am pretty sure we deleted the jsx for this style rule but in this case it still wouldn't break anything; just something to add to the code cleanup we need to do